### ISSUE TITLE
Mexico 5 meter DEM

### ISSUE THREAD

**Issue by nvkelso** [tilezen/joerd#136]
- http://www3.inegi.org.mx/sistemas/productos/default.aspx?cl=209010200

**Comment by iandees** [comment]
I've spent a couple hours trying to find the CRS for this data and haven't found anything that seems to work. Based on the "DATUM" attribute on an entry like [this one](http://www.beta.inegi.org.mx/app/biblioteca/ficha.html?upc=702825010717), I've been looking for information on `ITRF08`. There's entries like [this one](https://epsg.io/5332#) in epsg.io, but they don't produce an image in the right place on earth. I'm still looking for the right projection.

Once I do find the right projection, there are a couple changes I need to do to make the transcode step work:
- The [environment variables](https://github.com/mojodna/marblecutter/blob/mapzen/Dockerfile#L8) need to get updated to include the `.xyz` file type
- The [transcode.sh](https://github.com/mojodna/marblecutter/blob/d84c9293c747cbd22943ff47dbf257473134a14a/bin/transcode.sh) script needs to get updated to find a `.xyz` file inside a `.zip` file

**Comment by iandees** [comment]
Did a bunch of pre-processing (similar to the UK data) documented in this new repo: https://github.com/iandees/mx-lidar

#### Transcode files

```
aws s3 ls s3://elevation-sources-prod/mx_lidar/ --recursive | \
grep .tif$ | \
awk '{print $4}' | \
while read filename; do \
    bn=$(basename ${filename%.*}); \
    make submit-job job=aws/transcode-job.json.hbs input=s3://elevation-sources-prod/${filename} output=s3://elevation-sources-transcoded/$(dirname $filename)/${bn} name=${bn:0:50}
done
```

**Comment by iandees** [comment]
The transcoding finished and we ended up with 2,516 footprints in the database. This doesn't come close to covering Mexico, though:

![image](https://user-images.githubusercontent.com/261584/28117025-aa62ef8e-66d9-11e7-9246-43fccdbb1c66.png)

[This regex filter](https://github.com/iandees/mx-lidar/blob/master/list_urls.py#L23) in the URL listing step cut out most of the ~50k images from the portal. I think the rest of the images are "surface" (i.e. with buildings) images?

The data that did make it in looks pretty nice, though:

![image](https://user-images.githubusercontent.com/261584/28117182-5c2ed408-66da-11e7-96bd-be4bf8c62187.png)

**Comment by iandees** [comment]
I've poked around a bit on other pieces of the data at INEGI this morning and there's more to pull in, but almost none of it has valid projection information in a machine-readable format. I'm going to call it quits for now, as the next step (digging through 50,000 URLs to find a pattern in projection information) is vastly more time-consuming.

**Comment by nvkelso** [comment]
@iandees I am seeing Mexico data in the dev tiles (which looks good). Can you confirm this issue was completed in the v1.1 milestone, please?

**Comment by iandees** [comment]
Yep, there are 2,503 footprints in the database for Mexico, but as mentioned in [the comment above](https://github.com/tilezen/joerd/issues/136#issuecomment-314751249) it covers a very small part of Mexico.
